src/main/java/greencity/service/impl/PlaceServiceImpl.java

```diff
@@ -88,13 +84,34 @@ public Place save(PlaceAddDto dto, String email) {
      * @author Kateryna Horokh
      */
     private void setPlaceToOpeningHours(Place place) {
-        log.info("in setPlaceToOpeningHours(Place place) - {}", place.getName());
+        log.info(LogMessage.SET_PLACE_TO_OPENING_HOURS, place.getName());
+
         List<OpeningHours> hours = place.getOpeningHoursList();
-        hours.forEach(
-            h -> {
-                h.setPlace(place);
-                openingHoursService.save(h);
-            });
+        checkRepeatingValue(hours);
+        hours.stream()
+            .distinct()
+            .forEach(
+                h -> {
+                    h.setPlace(place);
+                    openingHoursService.save(h);
```

Comment:
check out transactions here.
If one openingHoursService.save(h); will fail, and the other one will be successful, will it save only successul 'saves' or none of them?